cmd/wdid/config.go

```diff
@@ -29,6 +29,9 @@ var defaultConfig = `
 [store]
 type = "bolt"
 file = "~/.config/wdid/wdid.db"
+
+[color]
+scheme = "default"
```

Comment:
this isn't being read yet, but is how the scheme will eventually be set (more permanent than cmd line arg)